kubectl/step7.md

```diff
@@ -1,6 +1,9 @@
-Create a deployment from this Kubernetes manifest. Review the manifest here
+Create a deployment from this Kubernetes manifest. You can review the manifest [here](https://raw.githubusercontent.com/kubernetes/website/master/content/en/examples/application/deployment.yaml).
```

Comment:
It would probably be nice here to explain the manifest a bit more to provide information to the user that will be used later on.